src/components/collection/Selector/Table/Hydrator.tsx

```diff
@@ -66,6 +70,7 @@ function Hydrator({ entity, selectedCollections }: TableHydratorProps) {
 
     return (
         <TableHydrator
+            disableQueryParamHack={disableQueryParamHack}
```

Comment:
Passing through the hack disable-er so that the Collections Selector pop up works AND the new beta transform create flow works.